recipes/configure.rb

```diff
@@ -82,6 +82,6 @@
 end
 
 service node['nrpe']['service_name'] do
-  action [:start, :enable]
+  action %i(start enable)
```

Comment:
This is another thing we disable in Rubocop because it actually makes it harder to understand what's going on

Reply:
Ok. ;)